distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -419,9 +419,9 @@ type_of value = @Builtin_Method "Meta.type_of"
 
    Arguments:
    - target: The value or type to get the attribute from.
-   - method_name: The name of the method or constructor to get the attribute for.
+   - method_name: The symbol representing method or constructor to get the attribute for.
    - parameter_name: The name of the parameter to get the attribute for.
-get_annotation : Any -> Text -> Text -> Any | Nothing
+get_annotation : Any -> Any -> Text -> Any | Nothing
```

Comment:
Can we do that?

```suggestion
get_annotation : Any -> Unresolved_Symbol -> Text -> Any | Nothing
```

Reply:
Not really, I am afraid. I tried, but`Unresolved_Symbol` object doesn't have a type. There is only `Meta.Unresolved_Symbol`, but that's not the right type - it is a wrapper around the real unresolved symbol.

TL;DR: There is no better type to use than `Any`, I am afraid.

Reply:
I see, ok.